src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/DefaultConnectionPlugin.java

```diff
@@ -10,7 +10,7 @@
 
   public DefaultConnectionPlugin(Log log) {
     if (log == null) {
-      throw new NullArgumentException("log");
+      throw new RuntimeException(NullArgumentException.getMessage("log"));
```

Comment:
Logger would be better.

Reply:
Done and changed in every place `log` is used.